### PR TITLE
add addons to crash report

### PR DESCRIPTION
**Pull request by RacoonDog**

### CODE REVIEW

**Comment by MineGame159** [comment]
Is this really needed? The crash report already contains all mods.

**Comment by RacoonDog** [comment]
> Is this really needed? The crash report already contains all mods.

yeah but it also contains all 50+ fapi modules
good to have imo

File: src/main/java/meteordevelopment/meteorclient/mixin/CrashReportMixin.java [hunk 1]
```diff
@@ -30,6 +32,31 @@ private void onAddStackTrace(StringBuilder sb, CallbackInfo info) {
             sb.append("Dev Build: ").append(MeteorClient.DEV_BUILD).append("\n");
         }
 
+        if (!AddonManager.ADDONS.isEmpty()) {
+            sb.append("\n[[ Addons ]]\n");
+            for (var addon : AddonManager.ADDONS) {
+                sb.append(addon.name);
+
+                List<String> authors = new ArrayList<>(List.of(addon.authors));
+                authors.removeIf(s -> s.equals("seasnail") || s.equals("seasnail8169"));
```
Comment (C10udburst) [inline src/main/java/meteordevelopment/meteorclient/mixin/CrashReportMixin.java:41]: :trollface:

Comment (RacoonDog) [reply src/main/java/meteordevelopment/meteorclient/mixin/CrashReportMixin.java:41]: id rather die standing than live kneeling

Comment (Wide-Cat) [reply src/main/java/meteordevelopment/meteorclient/mixin/CrashReportMixin.java:41]: o7

Comment (C10udburst) [reply src/main/java/meteordevelopment/meteorclient/mixin/CrashReportMixin.java:41]: o7

**Pull request closed without merging**